charts/fluentd-elasticsearch/values.yaml

```diff
@@ -70,6 +70,12 @@ elasticsearch:
     prefix: "logstash"
     prefixSeparator: "-"
     dateformat: "%Y.%m.%d"
+  ilm:
+    enabled: false
+    policy_id: logstash-policy
+    policy: {}
```

Comment:
maybe an example of  policy config would be good here

Reply:
I added an example policy config map as comment